src/MongoDB.Driver.Core/Core/Events/ConnectionCreatedEvent.cs

```diff
@@ -0,0 +1,87 @@
+﻿/* Copyright 2019-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using MongoDB.Driver.Core.Clusters;
+using MongoDB.Driver.Core.Configuration;
+using MongoDB.Driver.Core.Connections;
+using MongoDB.Driver.Core.Servers;
+
+namespace MongoDB.Driver.Core.Events
+{
+    /// <preliminary/>
```

Comment:
I realize this was copy/pasted from another event... but I just noticed it.

I didn't remember that `<preliminary>` even existed... and now that I see it wonder whether it belongs here. After all this event is from a spec... it's probably not so preliminary any more.

And what about removing `<preliminary>` from other events, either in this ticket or in a separate ticket.

Reply:
I will remove all these tags